### DISCUSSION TITLE
help configuring rspamd to be used from remote postfix

### DISCUSSION THREAD

**Discussion by klienn** [rspamd/rspamd#3727] [General]
Good day!
my postfix is in different host from my dovecot and rspamd, and everytime a mail comes the postfix will deliver it to my dovecot. Now I am having a hard time configuring my postfix to use the remote rspamd.
 
I have searched and found nothing.
Setting these:
```
smtpd_milters = inet:localhost:11332
non_smtpd_milters = inet:localhost:11332
milter_protocol = 6
milter_mail_macros =  i {mail_addr} {client_addr} {client_name} {auth_authen}
milter_default_action = accept
```
to 
```
smtpd_milters = inet:<ip where my rspamd is>:11332
non_smtpd_milters = inet:<ip where my rspamd is>:11332
milter_protocol = 6
milter_mail_macros =  i {mail_addr} {client_addr} {client_name} {auth_authen}
milter_default_action = accept
```
doesn't work
Any suggestions would be much appreciated

**Comment by fatalbanana** [comment]
Please use [discussions](https://github.com/rspamd/rspamd/discussions) (or [the mailing list, IRC, Telegram](https://rspamd.com/support.html)) for support questions.

The default listening address for all workers (including the proxy) is the localhost; have you [set the listening interface](https://rspamd.com/doc/quickstart.html#setting-listening-interface) appropriately? Can you connect to this service remotely? What goes wrong?

**Reply by klienn** [reply]
Good day, I am not sure if this is right but I set my `/etc/rspamd/local.d/worker-normal.inc` to `bind_socket = "202.60.9.10:11333";`.
Also, how  can I test connection to the service remotely? Thank you!

**Comment by moisseev** [comment]
Postfix should connect to a proxy worker. Instead of changing socket of normal worker you need to set `bind_socket = "202.60.9.10:11332";` or `bind_socket = "*:11332";` in the `etc/rspamd/local.d/worker-proxy.inc`
